app/dashboard/template.hbs

```diff
@@ -152,7 +152,25 @@
                                             <div class='col-sm-12 text-center'>
                                                 <p>{{t 'dashboard.quicksearch.no_projects.line1'}}</p>
                                                 <p>{{t 'dashboard.quicksearch.no_projects.line2'}}</p>
-                                                <img src='/assets/images/dashboard/quicksearch-min.png' alt='{{t 'dashboard.quicksearch.no_projects.preview_alt'}}' class='img-responsive center-block'>
+
+                                                <div local-class='iframe-container'>
+                                                    <iframe
```

Comment:
Here as well for the aria label and alt text. Do we have a screenshot of what happens when the video fails to load?